src/linpred.jl

```diff
@@ -167,7 +247,7 @@ function delbeta!(p::DensePredChol{T,<:CholeskyPivoted}, r::Vector{T}, wt::Vecto
     mul!(delbeta, transpose(p.scratchm1), r)
     # calculate delbeta = (X'WX)\X'Wr
     rnk = rank(p.chol)
-    if rnk == length(delbeta)
+    if rnk === length(delbeta)
```

Comment:
why this change? It should not matter, but I guess we want to check if values are equal, and do not care that their types are also equal.

Reply:
Got it. Thanks for your suggestion.